verifier/verifier.py

```diff
@@ -328,6 +353,39 @@ def verify_logprobs(
     return True, "", ""
 
 
+def verify_usage(
+    request: VerificationRequest, input_tokens: List[int]
+) -> Optional[Tuple[bool, str, str]]:
+    """Verify the usage information in the response."""
+    # Get actual token counts
+    actual_completion_tokens = len([item.token_id for item in request.output_sequence])
+    actual_total_tokens = len(input_tokens) + actual_completion_tokens
+
+    # Verify token counts
+    if request.usage.completion_tokens != actual_completion_tokens:
+        return (
+            False,
+            f"Reported completion tokens ({request.usage.completion_tokens}) does not match actual count ({actual_completion_tokens})",
+            "INVALID_COMPLETION_COUNT",
```

Comment:
We can put all three of these errors under the same enum, maybe something like "INCORRECT_USAGE_DATA"